src/mlpack/bindings/python/CMakeLists.txt

```
@@ -203,6 +203,9 @@ execute_process(COMMAND ${PYTHON_EXECUTABLE}
     "${PYTHON_INSTALL_PREFIX}"
     OUTPUT_VARIABLE CMAKE_PYTHON_PATH)
 string(STRIP "${CMAKE_PYTHON_PATH}" CMAKE_PYTHON_PATH)
+if (DEFINED ENV{PYTHONPATH})
+    string(APPEND CMAKE_PYTHON_PATH : $ENV{PYTHONPATH})
```

```suggestion
  string(APPEND CMAKE_PYTHON_PATH : $ENV{PYTHONPATH})
```
Tiny nit, two-space tabs. :)